.eslintrc.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "node": true,
+        "es2021": true
+    },
+    "extends": [
+        "eslint:recommended",
```

Comment:
isn't it better to pin those down ? to not get surprises ?

Reply:
It's a lot of rules that have to be defined explicitly and they're pretty universal, nothing opinionated.

Reply:
I'm just saying it might be changed from time to time, it doesn't have versions ?